src/components/Modal/src/ModalLayer.vue

```diff
@@ -126,6 +148,15 @@ export default {
 	background-color: rgba(0, 0, 0, 0.3);
 }
 
+.Transparent {
+	background-color: transparent;
+}
+
+.Hidden {
+	opacity: 0 !important;
```

Comment:
Is there a conflict with opacity somewhere that the important is necessary?

Reply:
Yes, the fade transition puts a `style: opacity` directly on the element so the `!important` is necessary. I can add an inline comment to explain this.